src/hotspot/share/runtime/arguments.cpp

```diff
@@ -2705,6 +2708,11 @@ jint Arguments::parse_each_vm_init_arg(const JavaVMInitArgs* args, bool* patch_m
     } else if (match_option(option, "-Xinternalversion")) {
       jio_fprintf(defaultStream::output_stream(), "%s\n",
                   VM_Version::internal_vm_info_string());
+      #ifdef INCLUDE_THIRD_PARTY_HEAP
+      jio_fprintf(defaultStream::output_stream(), "%s\n",
+                ThirdPartyHeap::version());
```

Comment:
In OpenJDK, arguments are indented to the opening bracket `'('`.
```suggestion
                  ThirdPartyHeap::version());
```